### PR TITLE
Update summer-school-2021.yaml

### PR DESCRIPTION
**Pull request by justjosie**
Re-ordering of the mosaic items for optimum fit - updating curriculum

### CODE REVIEW

**Review by techtolentino — APPROVED** [review]
Looks good @justjosie 

Thanks!

File: new-content/events/summer-school-2021.yaml [hunk 1]
```diff
@@ -28,44 +28,44 @@ mosaic:
...
+        url: 'https://qiskit.org/textbook/preface.html'
+        label: 'Read the textbook'
+        segment:
+          action: events-summer-school > mosaic > read-the-textbook
+
+    - position: 'second'
       title: 'IBM Quantum'
       description: 'IBM offers cloud access to the most advanced quantum computers available. Learn, develop, and run programs with our quantum applications and systems.'
-      image: '/images/events/seminar-series/mosaic-experts.png'
+      image: '/images/events/summer-school/IBM_System_One_display.jpg'
```
Comment (y4izus) [inline new-content/events/summer-school-2021.yaml:43]: Please, follow the same kebab-case style we are using for all the image names: `IBM_System_One_display` -> `ibm-system-one-display`

**Review by y4izus — CHANGES_REQUESTED** [review]
Just a comment on the code :)

Please, resolve the conflicts and we can merge

**Review by y4izus — APPROVED** [review]
Thanks!

**Pull request merged**